### PR TITLE
SLI-57 Binding a project (or changing binding) may make some issues o…

### PR DESCRIPTION
**Pull request by dbmeneses**
…utdated

### CODE REVIEW

**Comment by dbmeneses** [comment]
I had to change the messaging, so that the Issue View gets updated when the Issue store is changed instead of being updated when an analysis finishes.

File: src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectSettingsPanel.java [hunk 1]
```diff
@@ -82,13 +86,18 @@ public void save(SonarLintProjectSettings projectSettings) {
...
+      IssueStore store = project.getComponent(IssueStore.class);
+
+      console.info("Clearing all issues because binding changed");
+      store.clear();
     }
 
+  }
+
+  private boolean bindingChanged(SonarLintProjectSettings projectSettings) {
     if (projectSettings.isBindingEnabled() ^ bindPanel.isBindingEnabled()) {
```
Comment (henryju) [inline src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectSettingsPanel.java]: Is it not more readable to use != ?

**Comment by henryju** [comment]
LGTM

File: src/main/java/org/sonarlint/intellij/messages/IssueStoreListener.java [hunk 2]
```diff
@@ -25,8 +25,10 @@
 import java.util.Map;
 import org.sonarlint.intellij.issue.IssuePointer;
 
-public interface AnalysisResultsListener {
-  Topic<AnalysisResultsListener> SONARLINT_ANALYSIS_DONE_TOPIC = Topic.create("SonarLint New Issues", AnalysisResultsListener.class);
+public interface IssueStoreListener {
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/intellij/messages/IssueStoreListener.java]: MINOR Move constants to a class or enum. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1214)

File: src/main/java/org/sonarlint/intellij/ui/SonarLintIssuesPanel.java [hunk 3]
```diff
@@ -52,6 +52,7 @@
 import java.beans.PropertyChangeListener;
 import java.util.Collection;
 import java.util.Map;
+import java.util.Set;
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/intellij/ui/SonarLintIssuesPanel.java]: MINOR Remove this unused import 'java.util.Set'. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Pull request merged**